Tasks/JBossManagementCLI/task.json

```diff
@@ -13,18 +13,56 @@
     "demands": [],
     "version": {
         "Major": 1,
-        "Minor": 0,
+        "Minor": 112,
         "Patch": 0 
     },
     "instanceNameFormat": "Running management commands.",
     "inputs": [
       {
+        "name": "credsType",
+        "type": "pickList",
+        "label": "Authentication Method",
+        "defaultValue": "serviceEndpoint",
+        "required": true,
+        "helpMarkDown": "Use JBoss and WildFly service endpoint connection or enter connection credentials.",
+        "options": {
+            "serviceEndpoint": "JBoss and WildFly Connection",
+            "inputs": "Credentials"
+        }
+      },
+      {
         "name": "jbossEndpoint",
         "type": "connectedService:jbosseapwildflyendpoint",
         "label": "JBoss EAP/WildFly Endpoint",
         "defaultValue": "",
         "required": true,
-        "helpMarkDown": "Service endpoint for your JBoss EAP 7 / WildFly 8+ instance."
+        "helpMarkDown": "Service endpoint for your JBoss EAP 7 / WildFly 8+ instance.",
+        "visibleRule": "credsType = serviceEndpoint"
+      },
+      {
+        "name": "jbossServerUrl",
+        "type": "string",
+        "label": "JBoss EAP/WildFly Server URL",
+        "defaultValue": "",
+        "required": true,
+        "helpMarkDown": "URL for HTTP Management endpoint.",
+        "visibleRule": "credsType = inputs"
+      },
+      {
+        "name": "jbossManagementUser",
+        "type": "string",
+        "label": "Management User",
+        "defaultValue": "",
+        "required": true,
+        "visibleRule": "credsType = inputs"
+      },
+      {
+        "name": "jbossPassword",
+        "type": "string",
+        "label": "Password",
+        "defaultValue": "",
+        "required": true,
+        "visibleRule": "credsType = inputs"
```

Comment:
For the jbossPassword field, some suggested helpMarkDown:

"Enter the JBoss management user password. Use a new build variable with its lock enabled on the Variables tab to encrypt this value."